tests/Api/Request/RequestBuilderTest.php

```diff
@@ -168,13 +168,13 @@ public function buildProvider()
     /**
      * @dataProvider buildProvider
      *
-     * @param string $userIp
+     * @param string|bool $userIp
```

Comment:
Why is this a boolean when "No user ip provided"? Should be empty string.

Reply:
Because `false` is returned by `RequestBuilder::getUserIp`, when no IP exists

Reply:
Ok it's used as bool check as well, not the best implementation but we'll leave it for now.